wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php

```diff
@@ -106,7 +125,7 @@ function SSAPDFadminBuildPDF ()
 		switch_to_blog( $blogID );
 	}
 	
-	if ( current_user_can( 'create_and_download_pdf' ) ) //Only let them download the file if they are allowed to
+	if ( current_user_can( 'create_and_download_pdf' ) ) //Only let them download the file if they are admin
```

Comment:
Regarding the comment: anyone with the `create_and_download_pdf` can do so right? That doesn't necessarily have to be an admin